integration/test/ParseObjectTest.js

```diff
@@ -254,6 +254,14 @@ describe('Parse Object', () => {
     });
   });
 
+  it('cannot set an invalid date', (done) => {
+    let obj = new TestObject();
+    obj.set('when', new Date(Date.parse(null)));
+    Parse.Object.saveAll([obj]).fail((e) => {
```

Comment:
This actually succeeds , but failed the CI as the done() is never called.

Reply:
I just pushed this along with the other PR. This test passes after PR #495. So, we know that the unit test covers this fail condition. However, coverage is still down. What should I do?

Reply:
I should also mention that I copy pasted this test case from just above this unit test. That one invokes save instead of saveAll, and passes. It is the saveAll that was misbehaving.